timemachine/exchange/exchange_mover.py

```diff
@@ -61,12 +63,15 @@ def build_system(host_pdbfile: str, water_ff: str, padding: float):
     return solvated_host_system, solvated_host_coords, box, solvated_topology, nwa
 
 
-# currently un-used.
 def randomly_rotate_and_translate(coords, offset):
+    # coords: shape 3(OHH) x 3(xyz), water coordinates
     centroid = np.mean(coords, axis=0, keepdims=True)
     centered_coords = coords - centroid
     rot_mat = special_ortho_group.rvs(3)
-    rotated_coords = np.matmul(centered_coords, rot_mat)
+
+    # equivalent to standard form where R is first:
+    # (R @ A.T).T = A @ R.T
+    rotated_coords = centered_coords @ rot_mat.T
     return rotated_coords + offset
```

Comment:
question: should this be `+ offset + centroid`?

(aside: may be good to break this into a simpler function `randomly_rotate(coords)`, rather than passing through `randomly_rotate_and_translate(coords, offset)`)

Reply:
The intent is probably not well-documented. Like `randomly_translate`, the intent is to set the *centroid* of the displaced water to the location of "offset", a better name should be `new_loc` as opposed to `offset`.

Reply:
(will clarify in documentation and rename `offset`->`new_loc`)

Reply:
Ahh, thanks for clarifying this -- makes sense!